packages/client/tests/functional/tracing/tests.ts

```diff
@@ -510,72 +334,46 @@ testMatrix.setupTestSuite(({ provider }, suiteMeta, clientMeta) => {
         })
       })
 
-      const tree = await waitForSpanTree()
-
-      expect(cleanSpanTreeForSnapshot(tree)).toMatchSnapshot()
-
-      expect(tree.span.name).toEqual('prisma:client:transaction')
-      expect(tree.span.attributes['method']).toEqual('$transaction')
-      expect(tree.children).toHaveLength(3)
-
-      const create = (tree?.children || [])[0] as unknown as Tree
-      expect(create.span.name).toEqual('prisma:client:operation')
-      expect(create.span.attributes.model).toEqual('User')
-      expect(create.span.attributes.method).toEqual('create')
-
-      const findMany = (tree?.children || [])[1] as unknown as Tree
-      expect(findMany.span.name).toEqual('prisma:client:operation')
-      expect(findMany.span.attributes.model).toEqual('User')
-      expect(findMany.span.attributes.method).toEqual('findMany')
-
-      const itxRunner = (tree?.children || [])[2] as unknown as Tree
-      expect(itxRunner.span.name).toEqual('prisma:engine:itx_runner')
-
-      if (provider === 'mongodb') {
-        expect(itxRunner.children).toHaveLength(3)
+      let txQueries: Tree[] = []
 
-        return
+      if (provider !== 'mongodb') {
+        txQueries = [dbQuery(expect.stringContaining('BEGIN')), dbQuery('COMMIT')]
       }
 
-      expect(itxRunner.children).toHaveLength(5)
+      await waitForSpanTree({
+        name: 'prisma:client:transaction',
+        attributes: {
+          method: '$transaction',
+        },
+        children: [
+          operation('User', 'create', [clientSerialize()]),
+          operation('User', 'findMany', [clientSerialize()]),
+
+          {
+            name: 'prisma:engine:itx_runner',
+            attributes: { itx_id: expect.any(String) },
+            children: [
+              engineConnection(),
+              ...txQueries,
+              { name: 'prisma:engine:itx_query_builder', children: [...createDbQueries(false), engineSerialize()] },
+              { name: 'prisma:engine:itx_query_builder', children: [findManyDbQuery(), engineSerialize()] },
+            ],
+          },
+        ],
+      })
     })
   })
 
-  describe('tracing on $raw methods', () => {
+  describeIf(provider !== 'mongodb')('tracing on $raw methods', () => {
```

Comment:
It looks like we're missing a test for https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access#runcommandraw

Reply:
Can probably be done separately